### PR TITLE
Fix wrong import path in various files

### PR DESCRIPTION
**Pull request by nimrodshn**
Fix for https://github.com/MohawkTSDB/mohawk/issues/7.

cc @yaacov

### CODE REVIEW

File: mohawk.spec [hunk 1]
```diff
@@ -34,8 +34,8 @@ rm -rf vendor
 
 %build
 # set up temporary build gopath, and put our directory there
-mkdir -p ./_build/src/github.com/yaacov
-ln -s $(pwd) ./_build/src/github.com/yaacov/mohawk
+mkdir -p ./_build/src/github.com/MohawkTSDB
```
Comment (nimrodshn) [inline mohawk.spec]: Not sure about this one, should this be touched?

File: README.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Mohawk is a metric data storage engine, it's fun, fast, light and easy to use.
 
 ## Introduction
 
-[![Go Report Card](https://goreportcard.com/badge/github.com/yaacov/mohawk)](https://goreportcard.com/report/github.com/yaacov/mohawk)
+[![Go Report Card](https://goreportcard.com/badge/github.com/MohawkTSDB/mohawk)](https://goreportcard.com/report/github.com/MohawkTSDB/mohawk)
```
Comment (nimrodshn) [inline README.md]: @yaacov should this be changed as well?

**Comment by yaacov** [comment]
Thanks 👍

**Comment by yaacov** [comment]
> should this be touched?

Yes 🍰

**Pull request merged**